ci-operator/step-registry/hypershift/openstack/create/hypershift-openstack-create-chain.yaml

```diff
@@ -0,0 +1,208 @@
+    - default: "rhcos-latest-hcp-nodepool"
+      name: RHCOS_IMAGE_NAME
+    commands: |-
+      set -exuo pipefail
+      
+      HC_REGION=${HYPERSHIFT_AWS_REGION:-$LEASED_RESOURCE}
+      HC_ZONES="${HYPERSHIFT_HC_ZONES}"
+      if [[ -z "$HC_ZONES" ]]; then
+        HC_ZONES="$(oc get node -o jsonpath='{.items[0].metadata.labels.topology\.kubernetes\.io/zone}')"
+      fi
+      
+      RELEASE_IMAGE=${HYPERSHIFT_HC_RELEASE_IMAGE:-$RELEASE_IMAGE_LATEST}
+
+      if [ -f "${SHARED_DIR}/osp-ca.crt" ]; then
+        EXTRA_ARGS="${EXTRA_ARGS} --openstack-ca-cert-file ${SHARED_DIR}/osp-ca.crt"
+      fi
+
+      OPENSTACK_COMPUTE_FLAVOR=$(cat "${SHARED_DIR}/OPENSTACK_COMPUTE_FLAVOR")
+      OPENSTACK_EXTERNAL_NETWORK_ID=$(cat "${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK_ID")
+      
+      if [ ! -f "${SHARED_DIR}/clouds.yaml" ]; then
+          >&2 echo clouds.yaml has not been generated
+          exit 1
+      fi
+
+      # For disconnected or otherwise unreachable environments, we want to
+      # have steps use an HTTP(S) proxy to reach the API server. This proxy
+      # configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+      # environment variables, as well as their lowercase equivalents (note
+      # that libcurl doesn't recognize the uppercase variables).
+      if test -f "${SHARED_DIR}/proxy-conf.sh"
+      then
+        # shellcheck disable=SC1090
+        source "${SHARED_DIR}/proxy-conf.sh"
+      fi
+
+      CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)"
+      echo "$CLUSTER_NAME" > "${SHARED_DIR}/CLUSTER_NAME"
+      echo "$(date) Creating HyperShift cluster ${CLUSTER_NAME}"
+      COMMAND=(
+        /usr/bin/hypershift create cluster openstack
+        --name ${CLUSTER_NAME}
+        --node-pool-replicas ${HYPERSHIFT_NODE_COUNT}
+        --openstack-credentials-file ${SHARED_DIR}/clouds.yaml
+        --openstack-external-network-id ${OPENSTACK_EXTERNAL_NETWORK_ID}
+        --openstack-node-flavor ${OPENSTACK_COMPUTE_FLAVOR}
+        --openstack-node-image-name ${RHCOS_IMAGE_NAME}
+        --base-domain ${HYPERSHIFT_BASE_DOMAIN}
+        --control-plane-availability-policy ${HYPERSHIFT_CP_AVAILABILITY_POLICY}
+        --infra-availability-policy ${HYPERSHIFT_INFRA_AVAILABILITY_POLICY}
+        --pull-secret=/etc/ci-pull-credentials/.dockerconfigjson
+        --release-image ${RELEASE_IMAGE}
+        --annotations=hypershift.openshift.io/skip-release-image-validation=true
+      )
+      
+      if [[ $ENABLE_ICSP == "true" ]]; then
+        COMMAND+=(--image-content-sources "${SHARED_DIR}/mgmt_icsp.yaml")
+      fi
+      
+      if [[ -n $EXTRA_ARGS ]]; then
+        COMMAND+=(${EXTRA_ARGS})
+      fi
+     
+      if [[ $HYPERSHIFT_CREATE_CLUSTER_RENDER == "true" ]]; then
+        "${COMMAND[@]}" --render > "${SHARED_DIR}/hypershift_create_cluster_render.yaml"
+        exit 0
+      fi
+      
+      "${COMMAND[@]}"
+
+      set +e
```

Comment:
:grimacing:

Reply:
Leftover from:

https://github.com/openshift/release/blob/1771d8a973f464bdf44375c8ed392c5dff372b71/ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml#L107

I'll remove it.